content/docs/sidebar.json

```diff
@@ -139,7 +139,7 @@
         "label": "Experiment Management",
         "slug": "experiment-management",
         "source": "experiment-management/index.md",
-        "children": ["checkpoints"]
+        "children": ["experiments", "checkpoints", "organization"]
```

Comment:
organization -> Organizing Project ?
organization is way to abstract ... essentially it's about how do you structure the workflow? 

to be honest very hard to tell if the existing structure makes sense - would be great to see more content that you have in mind for each section

Reply:
I merged the **Organization Patterns** section back into the index so it has more context. If we get more content for this we can split and rename.